src/screens/auth/SignInScreen.tsx

```diff
@@ -0,0 +1,199 @@
+        {/* nightlight Logo */}
+        <NightlightLogoSvg style={SignInScreenStyles.logo} />
+
+        {/* Sign-In Inputs Container */}
+        <View style={SignInScreenStyles.inputsContainer}>
+          <Text style={SignInScreenStyles.inputsTitle}>Sign in with email</Text>
+          <TextInput
+            placeholder='Email'
+            placeholderTextColor={COLORS.DARK_GRAY}
+            style={[
+              SignInScreenStyles.emailInput,
+              isErrorVisible && SignInScreenStyles.errorInput,
+            ]}
+            autoCapitalize='none'
+            value={email}
+            onChangeText={setEmail}
+          />
+          <View style={SignInScreenStyles.passwordInputContainer}>
+            <TextInput
+              placeholder='Password'
+              placeholderTextColor={COLORS.DARK_GRAY}
+              style={[
+                SignInScreenStyles.passwordInput,
+                isErrorVisible && SignInScreenStyles.errorInput,
+              ]}
+              autoCapitalize='none'
+              secureTextEntry={!isPasswordVisible}
+              value={password}
+              onChangeText={setPassword}
+            />
+            <Pressable
+              onPress={togglePasswordVisibility}
+              style={SignInScreenStyles.passwordVisibilityButton}>
+              <Ionicons
+                name={`ios-eye${isPasswordVisible ? '' : '-off'}-outline`}
+                size={24}
+                color={COLORS.DARK_GRAY}
+              />
+            </Pressable>
+          </View>
+          <TouchableOpacity
+            onPress={handleForgotPasswordPress}
+            activeOpacity={0.75}
+            style={SignInScreenStyles.forgotPasswordLink}>
+            <Text style={SignInScreenStyles.forgotPasswordText}>
+              forgot password?
+            </Text>
+          </TouchableOpacity>
+        </View>
+
+        {/* Sign-In Button */}
+        <TouchableOpacity
+          onPress={handleSignInPress}
+          activeOpacity={0.75}
+          style={SignInScreenStyles.signInButton}>
+          <Text style={SignInScreenStyles.signInButtonText}>Sign In</Text>
+        </TouchableOpacity>
+
+        {/* Divider */}
+        <View style={SignInScreenStyles.signInOptionDividerContainer}>
+          <View style={SignInScreenStyles.signInOptionDividerLine} />
+          <Text style={SignInScreenStyles.signInOptionText}>
+            or continue with
+          </Text>
+          <View style={SignInScreenStyles.signInOptionDividerLine} />
+        </View>
+
+        {/* Sign in with Google */}
+        <TouchableOpacity
+          onPress={handleSignInWithGooglePress}
+          activeOpacity={0.75}
+          style={SignInScreenStyles.signInWithGoogleButton}>
+          {/* TODO: should we make this the actual Google logo? (idk abt copyright stuff) */}
+          <AntDesign
+            name='google'
+            size={20}
+            color={COLORS.DARK_GRAY}
```

Comment:
we have to use google's original color instead of darkening it
https://developers.google.com/identity/branding-guidelines